Library/Formula/ruby.rb

```diff
@@ -56,7 +58,7 @@ def install
       args << "--with-arch=#{Hardware::CPU.universal_archs.join(",")}"
     end
 
-    args << "--program-suffix=23" if build.with? "suffix"
+    args << "--program-suffix=#{program_suffix}" if build.with? "suffix"
```

Comment:
I guess not a deal-breaker. But feel free to shout at me, if I moved too quickly …